user/pages/03.Howtos/05.nova-rescue-mode/docs.en.md

```diff
@@ -148,7 +148,7 @@ $ sudo testdisk /dev/vdb1
 
 TestDisk 7.0, Data Recovery Utility, April 2015
 Christophe GRENIER <grenier@cgsecurity.org>
-http://www.cgsecurity.org
+https://www.cgsecurity.org
```

Comment:
Changing the quoted output of a 3rd party tool goes a bit far, don't you think?

Reply:
Ahah, true, I missed that!